tests/secrets/sanity/non_secrets/true_negative.json

```diff
@@ -0,0 +1,36 @@
+{
+    "upload_image_error_general": "Vi beklager, men det skjedde en feil ved opplasting. Sørg for at filen ikke overstiger {0} og at den er i et bildeformat (jpg, png, bmp eller gif).",
+    "my_profile": "Min profil",
+    "passwords_not_equal": "Passordene er ikke like.",
+    "invalid_one_time_code": "Du har ikke tastet en gyldig engangskode. Tast inn 4-sifret kode fra SMS.",
+    "access_key": "AKIAIOSFODNN7EXAMPLE",
+    "password_change_information": "For å bekrefte endringen, må vi sende deg kode på sms.",
+    "repeat_password": "Gjenta passord",
+    "password_with_length": "Passord (minst 6 tegn)",
+    "personal_information": "Personopplysninger",
+    "phone_number": "Telefonnummer",
+    "one_time_password": "Engangskode:",
+    "sendOneTimePassword": "Send engangskode",
+    "invalid_one_time_password": "Ugyldig engangskode",
+    "password_at_least_6_characters": "Passordet må ha minst 6 tegn.",
+    "password": "Passord",
```

Comment:
this line and the line below arent considered a secret?

Reply:
Not in the eyes of the base64 entropy detector :)